src/Altinn.App.Api/Infrastructure/Telemetry/IMetricFactory.cs

```diff
@@ -0,0 +1,58 @@
+using System.Diagnostics.Metrics;
+
+namespace Altinn.App.Api.Infrastructure.Telemetry;
+
+/// <summary>
+/// Provides a factory interface for creating metric meters.
+/// This interface extends the IMeterFactory to simplyfy creation
+/// </summary>
+public interface IMetricFactory : IMeterFactory
```

Comment:
I don't think we should implement `IMeterFactory` ourselves, we could just wrap it in the implementation (it handles the lifecycle of the underlying meters and such). I think our only goal with this is to let users not provide a name (as the name isn't really important in the way we use it)